src/components/Books.js

```diff
@@ -18,13 +18,13 @@ function Books() {
 
   return (
     <>
-      <div className="book-container">
+      <div className="container-fluid d-flex-column w-80 mt-5 book-container">
         {Object.keys(library.Booklist).map((bookItem) => (
 
-          <div className="book" key={uuidv4()}>
+          <div className="d-flex mt-3 book" key={uuidv4()}>
```

Comment:
- [ ] Kindly ensure that your bookstore book styling exactly matches the design you are given. Kindly add all the missing parts such as the progress bar 👍🏽.

Zeplin design 👇🏽 
![image](https://user-images.githubusercontent.com/63473584/201501759-2d30a0a2-322f-4855-8a94-29106f6b8904.png)

Your design 👇🏽 
![image](https://user-images.githubusercontent.com/63473584/201501787-94fd635c-e967-45fd-9287-969d21dcab84.png)